hw/ip/sram_ctrl/dv/sram_ctrl_base_sim_cfg.hjson

```diff
@@ -108,6 +108,11 @@
       uvm_test_seq: sram_ctrl_throughput_vseq
       run_opts: ["+zero_delays=1 +partial_access_pct=20"]
     }
+    {
+      name: "{name}_throughput_w_readback"
+      uvm_test_seq: sram_ctrl_throughput_vseq
+      run_opts: ["+zero_delays=1 +partial_access_pct=0 +init_w_readback=1"]
```

Comment:
I wonder whether we should follow up with something that turns the list into the form `["+a=b", "+c=d"]`. I understand that the difference doesn't matter, but it feels a bit silly to be giving "arguments with spaces" :-)

Reply:
Either way works for me  :-) Happy to change that if you think it makes sense. If we want to do this, in a separate PR as this affects some other entries as well?

Reply:
Eh, definitely a follow-up if it's worth doing (and not in this PR). I'm not completely sure it is! :-D